tests/test-local-dev-recipes.bicep

```diff
@@ -0,0 +1,202 @@
+      'Applications.Dapr/secretStores': {
+        default: {
+          templateKind: 'bicep'
+          plainHTTP: true
+          templatePath: 'reciperegistry:5000/recipes/local-dev/secretstores:latest'
+        }
+      }
+      'Applications.Datastores/sqlDatabases': {
+        default: {
+          templateKind: 'bicep'
+          plainHTTP: true
+          templatePath: 'reciperegistry:5000/recipes/local-dev/sqldatabases:latest'
+        }
+      }
+      'Applications.Dapr/stateStores': {
+        default: {
+          templateKind: 'bicep'
+          plainHTTP: true
+          templatePath: 'reciperegistry:5000/recipes/local-dev/statestores:latest'
+        }
+      }
+    }
+  }
+}
+
+resource app 'Applications.Core/applications@2023-10-01-preview' = {
+  name: 'local-dev-recipe-app'
+  location: 'global'
+  properties: {
+    environment: env.id
+    extensions: [
+      {
+          kind: 'kubernetesNamespace'
+          namespace: 'local-dev-recipe-app'
+      }
+    ]
+  }
+}
+
+resource webapp 'Applications.Core/containers@2023-10-01-preview' = {
+  name: 'mongodb-app-ctnr'
+  location: 'global'
+  properties: {
+    application: app.id
+    connections: {
+      mongodb: {
+        source: recipedb.id
+      }
+      daprpubsub: {
+        source: pubsubBroker.id
+      }
+      rabbitmq: {
+        source: rabbitmq.id
+      }
+      redis: {
+        source: redis.id
+      }
+      daprsecretstore: {
+        source: secretstore.id
+      }
+      sql: {
+        source: db.id
+      }
+      daprstatestore: {
+        source: statestore.id
+      }
+    }
+    container: {
+      image: magpieimage
+      env: {
+        CONNECTION_SQL_CONNECTIONSTRING: db.connectionString()
+      }
+      readinessProbe:{
+        kind:'httpGet'
+        containerPort:3000
+        path: '/healthz'
+      }
+    }
+    extensions: [
+      {
+        kind: 'daprSidecar'
+        appId: 'dpsb-recipe-app-ctnr'
+        appPort: 3000
+      }
+    ]
+  }
+}
+
+resource recipedb 'Applications.Datastores/mongoDatabases@2023-10-01-preview' = {
+  name: 'mongodb-db'
+  properties: {
+    application: app.id
+    environment: env.id
+    recipe: {
+      name: 'mongoazure'
+    }
+  }
+}
+
+resource pubsubBroker 'Applications.Dapr/pubSubBrokers@2023-10-01-preview' = {
+  name: 'dpsb-recipe'
+  properties: {
+    application: app.id
+    environment: env.id
+  }
+}
+
+resource rabbitmq 'Applications.Messaging/rabbitMQQueues@2023-10-01-preview' = {
+  name: 'rmq-recipe-resource'
+  properties: {
+    application: app.id
+    environment: env.id
+  }
+}
+
+resource redis 'Applications.Datastores/redisCaches@2023-10-01-preview' = {
+  name: 'rds-recipe'
+  properties: {
+    environment: env.id
+    application: app.id
+  }
+}
+
+resource secretstore 'Applications.Dapr/secretStores@2023-10-01-preview' = {
+  name: 'gnrc-scs-recipe'
+  properties: {
+    environment: env.id
+    application: app.id
+  }
+}
+
+resource mysecret 'core/Secret@v1' = {
```

Comment:
What is this secret store used for? Can we delete it?

Reply:
magpie container expects "mysecret" secret to be present https://github.com/radius-project/radius/blob/41231cb8f06256eb7283efd055e859d0cf8a0738/test/magpiego/bindings/daprsecretstore.go#L26-L29

I am creating a secret resource to achieve what we are doing here https://github.com/radius-project/radius/blob/aeb267d0f50e89a98b3ca9c41644ab75e7c08241/test/functional/daprrp/resources/dapr_secretstore_test.go#L116

Reply:
Ah gotcha, makes sense. Thanks!

Reply:
Could we move this up next to the container resource and add a comment describing why this is here?

Reply:
Ah wait it's related to the Dapr secret store. Nevermind. Thanks!